databases/backends/postgres.py

```diff
@@ -128,6 +128,9 @@ def __iter__(self) -> typing.Iterator:
     def __len__(self) -> int:
         return len(self._row)
 
+    def __str__(self) -> str:
+        return f"""{{{", ".join(f"'{key}': {value!r}" for key, value in self.items())}}}"""
```

Comment:
How about
```suggestion
        return str(dict(self))
```

The result should be the same.

Reply:
@vmarkovtsev imagine this scenario: 

A long list of Records is generated.  Each dict will be allocated and will quickly go out of scope but will remain until garbage collected. You create a spike in memory allocation which could cripple a low memory device (think embedded) and force it to swap.  

A long list of Records is a common and many IDEs will automatically call `__str__` on entries to show something useful in the debugger so I would imagine this scenario is not uncommon.

Reply:
> I am not familiar with the other DB backends - do they have similar __str__-s?

I wasn't concerned about other libraries when I wrote that method.  To me a typing.Mapping is better presented as a string similar to CPython's `dict.__str__()`.  It certainly makes more sense that mimicking CPython's `tuple.__str__()`.

Imagine this scenario.  Your row contains 12 integers.  If you can't remember the query string used, and the `tuple.__str__()` was mimicked you would likely have difficulty identifying which in is the one you are interested in.  Conversely, with the field names also supplied, the one you are interested in is obvious.